robot-db/initdata.script

```diff
@@ -0,0 +1,4 @@
+CREATE SCHEMA PUBLIC AUTHORIZATION DBA
```

Comment:
Each script should have a comment explaining why it was created. Presumably together with issue.